src/main.rs

```diff
@@ -237,120 +237,109 @@ fn format_duration(duration: Duration) -> Result<String> {
-                        .long("kernel")
-                        .value_name("BOOL")
-                        .help("Turn kernel info on or off.")
-                        .takes_value(true))
-                    .arg(Arg::with_name("uptime")
+                        .long("no-kernel")
+                        .help("Turn kernel version off.")
+                        .takes_value(false))
+                    .arg(Arg::with_name("no-uptime")
                         .short("u")
-                        .long("uptime")
-                        .value_name("BOOL")
-                        .help("Turn uptime info on or off.")
-                        .takes_value(true))
+                        .long("no-uptime")
+                        .help("Turn uptime info off.")
+                        .takes_value(false))
                     .arg(Arg::with_name("packages")
                         .short("p")
                         .long("packages")
-                        .value_name("BOOL")
-                        .help("Turn total package count on or off.")
-                        .takes_value(true))
+                        .help("Turn total package count on.")
+                        .takes_value(false))
                     .arg(Arg::with_name("music")
                         .short("m")
                         .long("music")
                         .value_name("SOURCE")
                         .help("Choose where to get music info. Supported options are \"mpd\" (mpc) and no (none).\n")
                         .takes_value(true))
-                    .arg(Arg::with_name("logo")
+                    .arg(Arg::with_name("no-logo")
                         .short("l")
-                        .long("logo")
-                        .value_name("BOOL")
-                        .help("Turn the logo (VALLEY) on or off.")
-                        .takes_value(true))
+                        .long("no-logo")
+                        .help("Turn the logo or ascii art off.")
+                        .takes_value(false))
                     .arg(Arg::with_name("logofile")
                         .short("L")
                         .long("logofile")
                         .value_name("FILE")
-                        .help("Specifies the file from which to read a custom ASCII logo.")
+                        .help("Specify the file from which to read a custom ASCII logo.")
                         .takes_value(true))
                     .arg(Arg::with_name("corners")
                         .short("C")
                         .long("corners")
                         .value_name("CHARACTER")
-                        .help("Specifies the corner style. Choose either \"■\" or \"0\".")
+                        .help("Specify the corner style. Choose either \"■\" or \"0\". Only used when corners are enabled.")
                         .takes_value(true))
                     .get_matches();
+    let current_user = Passwd::current_user();
```

Comment:
This will take some time to read `/etc/passwd`. You should only call it when the shell or user need to be displayed.

Reply:
I know, but I couldn't figure out an easy way on how to do that. What I wanted to do was: 

```Rust
if matches.is_present("no-user") || if matches.is_present("no-shell") {
    "do nothing"
} else {
    let current_user = Passwd::current_user();
}
```

The problem is, I couldn't figure out how to get that to work (in an efficient way) because "current_user" wouldn't exist outside of the if/else statement.

Reply:
There was this code, which you removed:

![image](https://user-images.githubusercontent.com/308347/57586808-f8bd0680-7503-11e9-8e5d-8e55a71ffb3f.png)

Reply:
Good point. I don't know what's with my brain today.